plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -574,9 +573,9 @@ var _ = Describe("validator", func() {
 					shoot.Spec.Maintenance.ConfineSpecUpdateRollout = ptr.To(confine)
 					shoot.Status.LastOperation = operation
 					if specChange {
-						shoot.Spec.Kubernetes.AllowPrivilegedContainers = ptr.To(
-							oldShoot.Spec.Kubernetes.AllowPrivilegedContainers == nil ||
-								!(*oldShoot.Spec.Kubernetes.AllowPrivilegedContainers))
+						shoot.Spec.Kubernetes.KubeControllerManager = &core.KubeControllerManagerConfig{
+							NodeMonitorGracePeriod: &metav1.Duration{Duration: 100 * time.Second},
+						}
```

Comment:
How does removal of `allowPrivilegedContainers` make new `kube-controller-manager` settings necessary?

Reply:
It could be any spec change. I just chose this.